### PR TITLE
Guide commands merryized

### PR DESCRIPTION
**Pull request by saraht45**
Merryized the following guide commands:
- !accfinger
- !finger
- !badname
- !badnames
- !goodname
- !broadcast
- !busy
- !grab
- !join
- !ungrab
- !whereis
- !dismiss
- !mute
- !unmute
- !office

Also restored the **who** command.

### CODE REVIEW

File: skoot/data/vault/Neoct/OOC/Verbs/who.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <Socials:SocObPreps/>
+        <Socials:SocObFilters/>
+      </Socials:SocialObject>
+    </Socials:SocialObjects>
+    <Socials:VerbActions/>
+    <Core:Properties>
+      <Core:Property property="merry:global:command">
+         X[M] /* Merryized +staff */
+
+/*
```
Comment (noahgibbs) [inline skoot/data/vault/Neoct/OOC/Verbs/who.xml:15]: Do we want to get rid of this commented-out code that did something different, but only for kargh?

**Comment by noahgibbs** [comment]
Other than the one minor thing, LGTM.

**Comment by noahgibbs** [comment]
LGTM. Ready to merge, then?

**Comment by saraht45** [comment]
> LGTM. Ready to merge, then?

Yep. Just did a test launch and nothing broke.

**Pull request merged**